GAD 8 - Binomial Heap/tests/binomilia/BinomialHeapTester.java

```diff
@@ -0,0 +1,2282 @@
+package tests.binomilia;
+
+import gad.binomilia.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * <i> Note: It is too difficult to create hard coded tests which test if the your binomial tree is a valid tree or not
+ * without actually comparing it with a proper implementation. <br>
+ * Therefore it is very likely that your insertions are correct, however my tests expect a different order. <br>
+ * My FastResult automatically sorts the nodes based on their ranks in decreasing order (and if the rank is same, based on their min values). <br>
+ * In that case you'll have to manually compare both the answers and manually go ahead! <br>
+ * Sorry for that but I don't have enough time to create perfect tests. If you want to improve it, you're gladly welcome to do so. <br>
+ * If I tried making an algorithm which actually compared if the tree is valid or not, it would be a very obfuscated recursive one
+ * which wouldn't have helped at all. However, with Strings it is very easy to visualise what it looks like, which is my motive behind the tests. <br>
+ * If you fail a test, @see <a href="http://webgraphviz.com">Graph visualiser</a> and compare the differences yourself. <br>
```

Comment:
@see Is a section of a Javadoc. You should put @see at the beginning of line 21 and simply use @link or the <a> Tag.

Reply:
Oh damn interesting, I didn't know about that, thanks for informing me about it, will fix it right away

Reply:
Fixed it